### DISCUSSION TITLE
Add --quiet as an intermediary log option between --silent and --verbose

### DISCUSSION THREAD

**Discussion by regisphilibert** [withastro/roadmap#1072] [Proposal]
### Body

# Summary

Introducing a new cli flag `--quiet` as an intermediary between `--silent` and `--verbose` which would slim down the default log by limiting individual file log to route files rather than html files. This proposal acts as a bump/revival of this [conversation](https://github.com/withastro/roadmap/discussions/400).

# Background & Motivation

Astro logs print every successful html file built which can be very helpful but bloats the build log and, on big projects, makes navigating to the critical info (usually at the bottom) a very long scroll. (especially on CI UX like Netlify or Cloudflare). Often, terminal apps will even cut the top of the log making it impossible to access the information relating to the start of the build.

# Goals

Slimming logs down by omitting files built and limiting it to the route file used and the amount of time spent on its generated files. Could also print the number of pages built.

# Example

Using `astro build`:

```
11:05:35 ▶ src/pages/[...section]/index.astro
11:05:35   ├─ /50-years/index.html (+6ms)
11:05:35   ├─ /about/index.html (+4ms)
11:05:35   └─ /take-action/index.html (+5ms)
11:05:35 ▶ src/pages/[...singleton].astro
11:05:35   ├─ /about/board-of-directors/index.html (+3ms)
11:05:35   └─ /staff/index.html (+6ms)
```

Using the new `astro build --quiet`
```
11:05:35 ▶ src/pages/[...section]/index.astro (+15ms) (3 files)
11:05:35 ▶ src/pages/[...singleton].astro (+9ms) (2 files)
```

**Comment by ematipico** [comment]
What about the other logs? What should we hide and what should we remove?

**Reply by regisphilibert** [reply]
I think the other logs are fine. The goal here is just to hide the numerous lines devoted to the names of the html files which take so much retail space and rarely matter when reading the log of a successful build.
